src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_NAME_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final String SUPPLY_STRING = "supply";
+    private static final String BUY_STRING = "buy";
+    private static final String RESULT_STRING = "result";
+    private static final String COMA_STRING = ",";
 
-    public void getStatistic(String fromFileName, String toFileName) {
+    private String[] readInfo(String fileName) {
+        File outputFile = new File(fileName);
```

Comment:
```suggestion
        File inputFile = new File(fileName);
```
When you read - data comes **IN** our program